python/pyramid/audit/csrf-check-disabled.yaml

```diff
@@ -0,0 +1,30 @@
+rules:
+  - id: pyramid-csrf-check-disabled
+    message: CSRF is disabled for this view. This is a security risk.
```

Comment:
Could you include another sentence that talks about the fix, similar to [here](https://github.com/returntocorp/semgrep-rules/blob/f0dfbdd231f6218aca034101b86fa3497f06890f/python/pyramid/audit/csrf-origin-check-disabled-globally.yaml#L13)?